packages/core/src/order/order.ts

```diff
@@ -66,6 +66,9 @@ export interface GatewayOrderPayment extends OrderPayment {
     mandate?: {
         id: string;
         url?: string;
+        mandateText?: {
```

Comment:
I think, you can rename it to `text` since in is in `mandate` object

```suggestion
        text?: {
```

Reply:
The structure that we receive from back-end is `payments.mandate.mandateText` where `mandateText` is
```
{
account_holder_name: 'Paypal-Ratepay',
iban: '43434344',
payment_reference: 'GD78789789',
}
```